src/pages/User/components/PullRequests/UserInfo/Message/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import getMessage from './getMessage';
+
+const Message = ({ pullRequestCount, otherReposCount }) => (
```

Comment:
Rename to `MotivationalMessage`

Reply:
Rename the file too.

`Message` is too generic and if should be able to be used everywhere which is not the case here.